client/src/components/categorical/value/index.js

```diff
@@ -21,20 +21,17 @@ import { AnnotationsHelpers } from "../../../util/stateManager";
 import { labelPrompt, isLabelErroneous } from "../labelUtil";
 
 /* this is defined outside of the class so we can use it in connect() */
-function _currentLabel(ownProps, categoricalSelection) {
-  const { metadataField, categoryIndex } = ownProps;
-  return String(
-    categoricalSelection[metadataField].categoryValues[categoryIndex]
-  ).valueOf();
+function _currentLabelAsString(ownProps) {
+  const { categorySummary, categoryIndex } = ownProps;
+  return String(categorySummary.categoryValues[categoryIndex]).valueOf();
```

Comment:
Seems like we can remove `.valueOf()`?

`String(categorySummary.categoryValues[categoryIndex]) === String(categorySummary.categoryValues[categoryIndex]).valueOf()`

Reply:
We want the primitive string type, not the String object.   If we remove the valueOf, we get the String() object.

Reply:
turns out that when the constructor is called as a function, it does return a primitive.  Only when used with the new operator does it return a String object.

So we can remove the valueOf call.